lte/gateway/c/oai/common/itti_free_defined_msg.c

```diff
@@ -143,8 +143,9 @@ void itti_free_msg_content(MessageDef* const message_p) {
       // DO nothing (trxn)
       break;
     case S11_PAGING_REQUEST: {
-      free((char*) message_p->ittiMsg.s11_paging_request.imsi);
-      message_p->ittiMsg.s11_paging_request.imsi = NULL;
+      if (message_p->ittiMsg.s11_paging_request.imsi) {
+        free_wrapper((void**) &message_p->ittiMsg.s11_paging_request.imsi);
+      }
```

Comment:
Paging request is also sent from HA task during AGW offload; During our discussion, we decided to keep both imsi and ue_ip address. In case imsi is not received in S11-PAGING request, mme shall look into ue_ip_imsi map to get the imsi and initiate paging

Reply:
yes, you are right. I think we need to comment on these caveats otherwise as you can see it is easy to forget the reason.

Reply:
Sure I will add comment